### PR TITLE
Fixed choosableSet remove #10

### PR DESCRIPTION
**Pull request by fmarinelli**

### CODE REVIEW

File: src/main/java/io/vertx/java/spi/cluster/impl/jgroups/domain/ImmutableChoosableSetImpl.java [hunk 1]
```diff
@@ -51,10 +51,12 @@ public ImmutableChoosableSetImpl(T value) {
   public ImmutableChoosableSet<T> remove(T value) {
     checkSanity(value);
 
-    if (value.equals(this.value)) {
+    if(this.value.equals(value)) {
```
Comment (purplefox) [inline src/main/java/io/vertx/java/spi/cluster/impl/jgroups/domain/ImmutableChoosableSetImpl.java]: Minor cosmetic detail - should have space between "if" and "("

File: src/main/java/io/vertx/java/spi/cluster/impl/jgroups/domain/MultiMapImpl.java [hunk 2]
```diff
@@ -27,19 +28,22 @@
 import java.util.Optional;
 import java.util.concurrent.ConcurrentHashMap;
 
-public class MultiMapImpl<K, V> implements Externalizable {
+public class MultiMapImpl<K, V> implements Externalizable, LambdaLogger {
 
   private final static Logger log = LoggerFactory.getLogger(MultiMapImpl.class);
+  private String name;
```
Comment (purplefox) [inline src/main/java/io/vertx/java/spi/cluster/impl/jgroups/domain/MultiMapImpl.java:34]: Should be final if it doesn't change.

**Comment by purplefox** [comment]
Looks good. Just some small comments above.

File: src/main/java/io/vertx/java/spi/cluster/impl/jgroups/domain/MultiMapImpl.java (hunk 2 above)
Comment (fmarinelli) [inline src/main/java/io/vertx/java/spi/cluster/impl/jgroups/domain/MultiMapImpl.java:34]: My problem is the Externalizable, I must have a no-args constructor

Comment (purplefox) [inline src/main/java/io/vertx/java/spi/cluster/impl/jgroups/domain/MultiMapImpl.java:34]: Ok, fair enough

**Comment by purplefox** [comment]
Feel free to merge

**Pull request merged**